docs-new/site/releases.md

```diff
@@ -0,0 +1,777 @@
+* [{{ icebergVersion }} Flink 1.17 runtime Jar](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-flink-runtime-1.17/{{ icebergVersion }}/iceberg-flink-runtime-1.17-{{ icebergVersion }}.jar)
+* [{{ icebergVersion }} Flink 1.16 runtime Jar](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-flink-runtime-1.16/{{ icebergVersion }}/iceberg-flink-runtime-1.16-{{ icebergVersion }}.jar)
+* [{{ icebergVersion }} Flink 1.15 runtime Jar](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-flink-runtime-1.15/{{ icebergVersion }}/iceberg-flink-runtime-1.15-{{ icebergVersion }}.jar)
+* [{{ icebergVersion }} Hive runtime Jar](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-hive-runtime/{{ icebergVersion }}/iceberg-hive-runtime-{{ icebergVersion }}.jar)
+
+To use Iceberg in Spark or Flink, download the runtime JAR for your engine version and add it to the jars folder of your installation.
+
+To use Iceberg in Hive 2 or Hive 3, download the Hive runtime JAR and add it to Hive using `ADD JAR`.
+
+### Gradle
+
+To add a dependency on Iceberg in Gradle, add the following to `build.gradle`:
+
+```
+dependencies {
+  compile 'org.apache.iceberg:iceberg-core:{{ icebergVersion }}'
+}
+```
+
+You may also want to include `iceberg-parquet` for Parquet file support.
+
+### Maven
+
+To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
+
+```
+<dependencies>
+  ...
+  <dependency>
+    <groupId>org.apache.iceberg</groupId>
+    <artifactId>iceberg-core</artifactId>
+    <version>{{ icebergVersion }}</version>
+  </dependency>
+  ...
+</dependencies>
+```
+
+## 1.3.1 release
+
+Apache Iceberg 1.3.1 was released on July 25, 2023.
+The 1.3.1 release addresses various issues identified in the 1.3.0 release.
+
+* Core
+  - Table Metadata parser now accepts null for fields: current-snapshot-id, properties, and snapshots ([\#8064](https://github.com/apache/iceberg/pull/8064))
+* Hive
+  - Fix HiveCatalog deleting metadata on failures in checking lock status ([\#7931](https://github.com/apache/iceberg/pull/7931))
+* Spark
+  - Fix RewritePositionDeleteFiles failure for certain partition types ([\#8059](https://github.com/apache/iceberg/pull/8059))
+  - Fix RewriteDataFiles concurrency edge-case on commit timeouts ([\#7933](https://github.com/apache/iceberg/pull/7933))
+  - Fix partition-level DELETE operations for WAP branches ([\#7900](https://github.com/apache/iceberg/pull/7900))
+* Flink
+  - FlinkCatalog creation no longer creates the default database ([\#8039](https://github.com/apache/iceberg/pull/8039))
+
+## Past releases
+
+### 1.3.0 release
+
+Apache Iceberg 1.3.0 was released on May 30th, 2023.
+The 1.3.0 release adds a variety of new features and bug fixes.
+
+* Core
```

Comment:
I think this is not formatted correctly when I deploy locally.

<img width="732" alt="image" src="https://github.com/apache/iceberg/assets/6235869/fd2ea1b7-d9c2-432b-81ca-0bc96b0bd762">

Reply:
Great catch @aokolnychyi!

Looks like the markdown Hugo uses isn't as picky with indentation.

Here's the 1.3.1 release markdown.

```
* Core
  - Table Metadata parser now accepts null for fields: current-snapshot-id, properties, and snapshots ([\#8064](https://github.com/apache/iceberg/pull/8064))
* Hive
  - Fix HiveCatalog deleting metadata on failures in checking lock status ([\#7931](https://github.com/apache/iceberg/pull/7931))
```

Which [results in what you've shown above](https://bitsondatadev.github.io/making-iceberg-docs/releases/#131-release). 

I checked if there was a change after we moved away from mkdocs previously and keeps rendering incorrectly through 0.13.0:

```
**High-level features:**

* **Core**
  * Catalog caching now supports cache expiration through catalog property `cache.expiration-interval-ms` [[\#3543](https://github.com/apache/iceberg/pull/3543)]
```

Which still [renders incorrectly](https://bitsondatadev.github.io/making-iceberg-docs/releases/#0130).

0.12.0 on the other hand, renders correctly and it seems the indentation fixes it.

```
**High-level features:**

* **Core**
    * Allow Iceberg schemas to specify one or more columns as row identifiers [[\#2465](https://github.com/apache/iceberg/pull/2465)]. Note that this is a prerequisite for supporting upserts in Flink.
```

Which [renders what I believe is expected](https://bitsondatadev.github.io/making-iceberg-docs/releases/#0120).

I would propose instead that we follow the 0.14.0 release format that makes the section title H4 elements.

```
### 0.14.0 release

Apache Iceberg 0.14.0 was released on 16 July 2022.

#### Highlights

* Added several [performance improvements](#performance-improvements) for scan planning and Spark queries
* Added a common REST catalog client that uses change-based commits to resolve commit conflicts on the service side
```

Which [renders properly](https://bitsondatadev.github.io/making-iceberg-docs/releases/#0140-release) as it is standard compatible markdown. Thoughts?

Reply:
Also, this will wait until the next phase. There are a lot of issues like this that I'd like to address in a separate PR.